deploy/instance/docker-compose.base.yaml

```
@@ -0,0 +1,57 @@
+  services-auth:
+    image: ghcr.io/sderickson/saf-services-auth:latest
+    volumes:
+      - ../../dbs/auth/data:/app/dbs/auth/data
+    build:
+      context: ../..
+      dockerfile: services/auth/Dockerfile
+    environment:
+      - NODE_ENV=${NODE_ENV}
+      - PROTOCOL=${PROTOCOL}
+      - DOMAIN=${DOMAIN}
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)